code/drasil-lang/Language/Drasil/Expr/Extract.hs

```diff
@@ -12,6 +12,7 @@ deNames (AlgebraicExpr e)  = eNames e
 deNames (SpaceExpr _)      = []
 deNames (BinOp _ l r)      = deNames l ++ deNames r
 deNames (AssocBinOp _ des) = concatMap deNames des
+deNames (ForAll u _)      = []
```

Comment:
Do we really want to collect "u"? In some way, it's a bound name. It depends on what it's used for.

Reply:
Ah, you're right! This is ultimately used for extracting/collecting UIDs to place in both the table of symbols and table of units, so it wouldn't need to be. It might be needed for the table of units if we find any issues later, but it probably shouldn't be put into the table of symbols, I imagine.